Vic2ToHoI4/Source/OutHoi4/OutHoi4Country.cpp

```diff
@@ -552,6 +552,10 @@ void outputResearchSlots(std::ostream& output, const bool& greatPower, const boo
 	{
 		output << "set_research_slots = 3\n";
 	}
+	else if (unrecognized)
+	{
+		output << "set_research_slots = 0\n";
```

Comment:
Game is fine with this? They actually have 0?

Reply:
Yep! Works perfectly. Between this and the empty focus trees these nations are comfortably useless.